docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,108 @@
+# Create Flink job clusters with Helm Chart
+
+## Overview
+
+This Helm Chart is an addition to the [existing](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/config/samples) way of deploying Flink job clusters.
```

Comment:
Mention the benefit provided by this Helm Chart over creating a YAML directly, so that users can decide which method to use.

Reply:
True! I'll make sure to add it.